pages/getting-started/developers.md

```diff
@@ -192,6 +192,14 @@ $medium-screen: 620px !default;
 $large-screen:  1120px !default;
 ```
 
+### Set the base asset path (fonts and images)
+If you copy the USWDS `dist` directory to a folder on your project or your fonts and images are in the same folder, set the location of the directory with `$asset-path: path/to/my/assets/`. The default is set to `$uswds-path: ../`. This will be appended to the font and image path variables so you only need to set this variable. You can also set the font and image paths individually:
```

Comment:
Should `$uswds-path: ../` be `$asset-path: ../`? I don't think `$uswds-path` exists, does it?

Reply:
@hursey013 yes, that should have been matching the other variable. Fixed in 4544fbb.